ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/CompactMessage.qml

```diff
@@ -54,6 +54,14 @@ Item {
         anchors.leftMargin: chatTextItem.chatHorizontalPadding
         anchors.right: parent.right
         anchors.rightMargin: chatTextItem.chatHorizontalPadding
+
+        Connections {
+            enabled: !appSettings.compactMode
```

Comment:
This connection will never be used in this component. If compactMode is on, then enabled is `false`. if compactMode is fasle, then is component is not loaded at all, since it is the CompactMessage 😛